### PR TITLE
Update engine.py

### PR DESCRIPTION
**Pull request by os0000os**
implements accuracy calculation in both train and evaluate functions

### CODE REVIEW

File: src/engine.py [hunk 1]
```diff
@@ -16,24 +16,27 @@ def train_one_epoch(model: nn.Module, iterator: Iterable,
 
     model.train()
     for image, label in iterator:
-        image = image.to(device)
-        label = label.to(device)
+        image = image.to(device) # [BATCH_SIZE=256, RGB=3, IMAGE_SIZE=32, IMAGE_SIZE=32]
```
Comment (niniack) [inline src/engine.py:19]: is the input dim 32 for resnet typically? or is this referring to something else?

Comment (ab7289) [reply src/engine.py:19]: i believe it's just for CIFAR10, normally I think the input size for resnets is 224 (i think)

**Review by ab7289 — APPROVED** [review]
approved
